packages/datetime/package.json

```diff
@@ -33,7 +33,7 @@
     "@blueprintjs/core": "^2.0.0-rc.2",
     "classnames": "^2.2",
     "react-day-picker": "^7.0.7",
-    "tslib": "^1.5.0"
+    "tslib": "^1.9.0"
```

Comment:
I think all of these need to be `~1.9.0`, similar to the typescript version range

Reply:
i don't think so, it's really a minimum for the new functions added to support `esModuleInterop`

Reply:
Hmm, I thought that the behavior of these functions can change across minor versions... but that doesn't seem to be the case looking at its README. We do it for internal libraries, but this should be fine; we should probably switch over to a `^` range

Reply:
switch what over to `^` range?

makes sense to leave typescript with `~` to allow patches but exclude minors (cuz those can change things significantly)

Reply:
sorry, I meant switch our internal libraries which use `"tslib": "~` over to `^`. unrelated to this PR.